src/assets/i18n/de.json

```diff
@@ -1682,6 +1684,12 @@
     "remove_users_car_error": "Error occurred while removing the users",
     "car_not_found": "Car has not been found",
     "car_error": "Error occurred while retrieving the car",
+    "dialog": {
```

Comment:
Try to avoid this tree of naming, it makes difficult to search for a string.
I would prefer delete_title + delete_confirm key without dialog.